build/CodeSign/certificate.ignore

```diff
@@ -3,6 +3,9 @@ obj\*, Ignore Intermediates
 bin\*\instrumentationengine.lib.tests.dll, Ignore test assembly
```

Comment:
Not that I'm asking you to do this, but I wonder if we should move all test artifacts under a Tests folder and then just exclude the folder rather than excluding individual files.

Reply:
Absolutely agree